.github/workflows/release.yml

```diff
@@ -12,27 +12,35 @@ jobs:
   linux:
     name: Build - Linux
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - name: linux-aarch64
+            target: aarch64-unknown-linux-gnu
+          - name: linux
+            target: x86_64-unknown-linux-gnu
     steps:
       - name: Check out code
         uses: actions/checkout@v3
-      - name: Set up docker buildx
-        uses: docker/setup-buildx-action@v2
-      - name: Build docker image
-        uses: docker/build-push-action@v3
-        with:
-          cache-from: type=gha
-          cache-to: type=gha,mode=max
-          context: ./ci/docker
-          push: false
-          load: true
-          tags: volta
-      - name: Compile and package Volta
-        run: docker run --volume ${PWD}:/root/workspace --workdir /root/workspace --rm --init --tty volta /root/workspace/ci/build-linux.sh volta-linux
+      - uses: actions-rs/toolchain@master
+        with:
+          profile: minimal
+          toolchain: 1.63
+          override: true
+      - name: Cargo Cache
+        uses: Swatinem/rust-cache@v2
+      - name: Install cross
+        run: cargo install cross
+      - name: Build
+        run: cross build --release --target ${{ matrix.target }}
+      - name: Create artifact
+        run: tar -czvf volta-${{ matrix.name }}.tar.gz volta volta-shim volta-migrate
```

Comment:
I'm not on a machine right now where I can cleanly check, but does this change the `glibc` dependency for the standard linux build? The Docker system setup right now is built intentionally to support running on an old version of `glibc`.

For the ARM build, since we haven't supported it in the past, it's not a big deal to use a more recent version, but for the existing linux build we've been committed to backwards compatibility.

Reply:
I have never done any development in Rust so it would take me a long time to do a good analysis but I can try to make a proposal.

Could [QEMU](https://github.com/docker/setup-qemu-action) be used in Github actions with a matrix to build 2 linux builder images (`linux/amd64` and `linux/arm64` platform) with differents tags and then use these images with the `docker run --platform` flag to build volta for both architectures?